ci/coq-tests.el

```diff
@@ -245,15 +269,43 @@ For example, COMMENT could be (*test-definition*)"
 
 (ert-deftest 070_coq-test-regression-wholefile-no-proof ()
   "Regression test for no proof bug"
-  (coq-fixture-on-file
-   (coq-test-full-path "test_wholefile.v")
+  (coq-fixture-on-file 
+   (coq-test-full-path "test_wholefile.v") 
    (lambda ()
      (proof-process-buffer)
      (proof-shell-wait)
      (goto-char (point-min))
      (insert "(*.*)")
      (should (equal (proof-queue-or-locked-end) 1)))))
 
+
+(ert-deftest 080_coq-test-regression-show-proof-stepwise()
+  "Regression test for the \"Show Proof\" option"
+    (coq-fixture-on-file 
+   (coq-test-full-path "test_stepwise.v") 
+   (lambda ()
+     (coq-test-goto-before " (*test-insert*)")
+     (proof-goto-point)
+     (proof-shell-wait)
+     (coq-should-response "(fun (A : Prop) (proof_of_A : A) => ?Goal)"))
```

Comment:
then maybe you could use here `(coq-should-buffer-string "(fun …` (see previous review comment)